hashintel/hash
# Print logs on unexpected output

## 🌟 What is the purpose of this PR?

When the test outputs number is different than the expected we want to be able to understand why.
This PR adds another test run when we run into that case and prints the resulting logs.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1199548034582004/1202206648648215/f) _(internal)_

## 🔍 What does this change?

- Run the test a second time with the most verbose log level and print the logs at the end

## 📜 Does this require a change to the docs?

No

## 🛡 What tests cover this?

None

## ❓ How to test this?

1.  Checkout the branch / view the deployment
2.  Try changing the number of expected outputs from one of the test in `tests/examples`
3.  Confirm that `cargo run <name of the test>` prints the log
